### PR TITLE
add missing security rules about NULL check to doc

### PR DESCRIPTION
**Pull request by icy17**
##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man3/SSL_CTX_set_verify.pod
Comment (InfoHunter) [inline doc/man3/SSL_CTX_set_verify.pod:35]: This is not consistent with other uppercase "MUST NOT" phrases.

File: doc/man3/EVP_EncryptInit.pod
Comment (InfoHunter) [inline doc/man3/EVP_EncryptInit.pod:364]: No trailing space.

File: doc/man3/ASN1_STRING_length.pod
Comment (InfoHunter) [inline doc/man3/ASN1_STRING_length.pod:31]: I checked some other places among the pod files, NULL should also be bold as "B\<NULL>"

**Comment by icy17** [comment]
Thanks for the feedback. I'll fix the issues.

File: doc/man3/ASN1_STRING_length.pod
Comment (t8m) [reply doc/man3/ASN1_STRING_length.pod]: No! It should not.

File: doc/man3/ASN1_STRING_length.pod
Comment (t8m) [inline doc/man3/ASN1_STRING_length.pod:31]: Please revert the `NULL`->`B<NULL>` changes. @InfoHunter was wrong, NULL should not be put in bold.

**Review by t8m — CHANGES_REQUESTED** [review]

**Review by t8m — APPROVED** [review]

**Comment by FdaSilvaYY** [comment]
Typo : `rulse` in Commit Message

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
ping @openssl/committers for the second review

**Comment by beldmit** [comment]
BTW, why do we have ABI label for doc-only changes?..

**Review by nhorman — APPROVED** [review]

**Review by shahsb — APPROVED** [review]
LGTM.

**Comment by t8m** [comment]
Merged to the master branch. Thank you for your contribution.

**Pull request closed without merging**
